CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## [0.2.0] - 2024/10/07
+### Lib
```

Comment:
For some reason the previous entry in the changelog used the `Lib` label, which is wrong. We should use the standard semantic commit labels: "add", "update", etc.

Reply:
If in doubt, check the changelog of video-core: https://github.com/newrelic/video-core-js/blob/master/CHANGELOG.md